Frameworks/CoreGraphics/CGContextCairo.mm

```diff
@@ -1629,6 +1630,229 @@
     UNLOCK_CAIRO();
 }
 
+void CGContextCairo::_CGContextDrawStrokedPath(cairo_path_t* flatPath, int i, int prev, int total, CGFloat lineWidth) {
```

Comment:
Null check is not required here as the path is always obtained from cairo which guarantees returning a valid pointer, but a check for flatPath->status would be nice to have if for some reason the cairo context used to retrieve the path was in an error state

---
In reply to: [70112840](https://github.com/Microsoft/WinObjC/pull/625#discussion_r70112840) [](ancestors = 70112840)